adapters/overnight/src/index_json_file.ts

```diff
@@ -0,0 +1,71 @@
+import BigNumber from "bignumber.js";
+import { CHAINS, PROTOCOLS, SNAPSHOTS_BLOCKS } from "./sdk/config";
+import { getLPValueByUserAndPoolFromPositions, getPositionsForAddressByPoolAtBlock } from "./sdk/subgraphDetails";
+
+(BigInt.prototype as any).toJSON = function () {
+  return this.toString();
+};
+
+import fs from 'fs';
+
+interface LPValueDetails {
+  pool: string;
+  lpValue: string;
+}
+
+interface UserLPData {
+  totalLP: string;
+  pools: LPValueDetails[];
+}
+
+interface OutputData {
```

Comment:
Can you please make sure the output data follows the below schema mentioned in readme.
https://github.com/delta-hq/l2-lxp-liquidity-reward#output-data-schema

Reply:
Now should be done!

Reply:
is still smth wrong? @nitish-91